internal/cli/encrypt.go

```diff
@@ -33,16 +33,19 @@ func newEncryptCmd() *cobra.Command {
 }
 
 func runEncryptCmd(cmd *cobra.Command, args []string) error {
-	logger := log.NewSimpleLogger(verbose)
-
 	encrypter, err := encrypt.NewEncrypter(
 		encrypt.WithParts(keyParts),
 		encrypt.WithThreshold(keyThreshold),
```

Comment:
I was thinking that it will be nice to have a `WithLogger` opt to set the logger in the `Encrypter`, what do you think?

Reply:
Agree with you. Do you think `runEnctryptCmd(...)` is the right place to chose the choosen logger?
I'm thinking on factory function in _github.com/enrichman/stegosecrets/internal/log/logger.go_ that will accept silent and verbose flags.

There we could handle the execution with `-s` and `-v` passed at same time

Reply:
I'm not sure on how the factory will look like, I don't miss my old java days. 😆